examples/DocQA/app.py

```diff
@@ -0,0 +1,300 @@
+import asyncio
+import json
+import os
+import uuid
+from queue import Queue
+from threading import Thread
+from typing import AsyncGenerator, Generator, List, Optional
+
+import chromadb
+import gradio as gr
+import requests
+from chromadb.utils import embedding_functions
+from dotenv import load_dotenv
+from llama_stack_client import LlamaStackClient
+from llama_stack_client.lib.agents.agent import Agent
+from llama_stack_client.lib.agents.event_logger import EventLogger
+from llama_stack_client.types.agent_create_params import AgentConfig
+from llama_stack_client.types.memory_insert_params import Document
+import re
+
+
+# Load environment variables
+load_dotenv()
+
+HOST = os.getenv("HOST", "localhost")
+PORT = int(os.getenv("PORT", "5000"))
```

Comment:
I think this should be `LLAMA_STACK_PORT` to make it clearer

Reply:
Sure, this has been changed